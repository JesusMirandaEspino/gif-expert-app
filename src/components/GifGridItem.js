import React from 'react'
import PropTypes from 'prop-types';
export default function GifGridItem( { id, title, url } ) {



    return (
        <section className="fila" >

            <div className="contenedor1" >

                <div className="col-full-12" >

                    <div className="card  animate__animated animate__fadeIn">
                        <img  src={ url} title={  title }   alt={ title } />
                        <p>{ title }</p> 
                    </div>


                </div>

            </div>
            
        </section  >
    )
}




GifGridItem.propTypes = { 
    title: PropTypes.string.isRequired,
    url: PropTypes.string.isRequired
}
