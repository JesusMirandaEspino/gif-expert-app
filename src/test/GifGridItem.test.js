import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import GifGridItem from '../components/GifGridItem';



describe( 'Pruebas en el archivo <GifGridItem />', () => { 

        const title = 'Es un titulo';
        const url = 'https://localhostesunaurl';

    test( 'Debe mostrar el componente correctamente', () => { 

        const wrapper = shallow( <GifGridItem  title={title} url={url} /> );


        expect( wrapper ).toMatchSnapshot();

    });




} );


