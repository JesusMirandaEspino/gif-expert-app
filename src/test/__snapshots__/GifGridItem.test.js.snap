// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el archivo <GifGridItem /> Debe mostrar el componente correctamente 1`] = `
<section
  className="fila"
>
  <div
    className="contenedor1"
  >
    <div
      className="col-full-12"
    >
      <div
        className="card  animate__animated animate__fadeIn"
      >
        <img
          alt="Es un titulo"
          src="https://localhostesunaurl"
          title="Es un titulo"
        />
        <p>
          Es un titulo
        </p>
      </div>
    </div>
  </div>
</section>
`;
